Source/BoundaryConditions/WarpXFieldBoundaries.cpp

```diff
@@ -131,8 +135,10 @@ void WarpX::ApplyBfieldBoundary (const int lev, PatchType patch_type, DtType a_d
     if (lev == 0) {
         if (a_dt_type == DtType::FirstHalf) {
             if(::isAnyBoundary<FieldBoundaryType::Absorbing_SilverMueller>(field_boundary_lo, field_boundary_hi)){
+                auto Efield_fp_new = m_fields.get_mr_levels_alldirs("Efield_fp",max_level); // JRA, new to prevent shadow
+                auto Bfield_fp = m_fields.get_mr_levels_alldirs("Bfield_fp",max_level);
```

Comment:
Can this be renamed back to `Efield_fp` (without `new`) now that `Efield_fp` has been removed from WarpX.H?
```suggestion
                auto Efield_fp = m_fields.get_mr_levels_alldirs("Efield_fp",max_level); // JRA, new to prevent shadow
                auto Bfield_fp = m_fields.get_mr_levels_alldirs("Bfield_fp",max_level);
```